src/App.tsx

```diff
@@ -1,8 +1,112 @@
-import React from 'react';
+import React, { useState, FC } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { getData } from './api/api';
 import './App.css';
+import { Todo } from './interfaces/TodoInterface';
+import { User } from './interfaces/UserInterface';
+import { TodoList } from './components/TodoList/TodoList';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setLoading] = useState<boolean>(false);
```

Comment:
do not specify primitive types 
```suggestion
  const [isLoading, setLoading] = useState(false);
```